.github/workflows/msbuild.yml

```diff
@@ -1,6 +1,6 @@
 name: MSBuild
 
-on: push
+on: pull_request
```

Comment:
Should we add a `types: [opened, synchronize]` so that we do not trigger the build on any type of change to the pull request.

Reply:
I was not aware, but it looks like the defaults are ok https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows 

>  By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.